maraboupy/Marabou.py

```diff
@@ -42,7 +42,7 @@ def read_nnet(filename, use_nlr=False):
     return MarabouNetworkNNet(filename, use_nlr=use_nlr)
 
 
-def read_tf(filename, inputNames=None, outputName=None, savedModel=False, savedModelTags=[]):
+def read_tf(filename, inputNames=None, outputName=None, savedModel_v1=False, savedModel_v2=False, savedModelTags=[]):
```

Comment:
is it possible to merge saveModel_v1, saveModel_v2 as a single argument called for instance "modelFormat". which is a string that can be one of  "frozen", "savedModelv1", "savedModelv2"?

Reply:
Yes I think that's a better way to do it. I'll work on those changes. I'm also working on creating tests for savedModelv1, so there's still a bit of work to do before this is ready to be merged.